packages/validator-ajv8/package.json

```diff
@@ -27,13 +27,13 @@
     ]
   },
   "dependencies": {
-    "ajv": "^8.11.0",
+    "ajv8": "npm:ajv@^8.11.0",
```

Comment:
Part of the reason the dev server was crashing was because Vite was resolving `'ajv'` to Ajv 6 in `'@rjsf/validator-ajv8'`.

Will this affect anything for users?

Reply:
Maybe switch things in the `ajv6` validator instead?

Reply:
The fix doesn't work if I just change it for ajv6, not sure why.